app/assets/javascripts/constants/api_constants.es6.jsx

```diff
@@ -1,13 +1,14 @@
 (() => {
   class ApiConstants {
-    
+
     get students() {
       return {
         index: '/api/students',
+        unmatched: '/api/students/unmatched',
         delete: (id) => `/api/students/${id}`,
         update: (id) => `/api/students/${id}`,
         show: (id) => `/api/students/${id}`,
-        unmatched: '/api/students/unmatched',
+        upcoming_lessons: (id) => `/api/students/upcoming_lessons/${id}`,
```

Comment:
can you also refactor the places where Apiconstants.teachers.possible_teachers is used